### ISSUE TITLE
question about Genus name

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#535]
[Originally posted on GoogleCode (id 102) on 2010-12-19]

<b>(This is the template to report a data issue. If you want to report</b>
<b>an interface issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/3245

<b>What data are incorrect or missing?</b>
Hieracium ×floribundum Wimmer &amp; Grabowski 

<b>What data are you expecting instead?</b>
Pilosella ×floribunda (Wimm. &amp; Grab.) Arvet-Touvet.
Since this hybrid is shown to have two parents in the genus Pilosella, shouldn’t the hybrid also be a Pilosella?  

<b>If applicable, please provide an authoritative source.</b>
IPNI

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2010-12-20 19:35Z]

It is not clear that Pilosella floribunda Arvet-Touvet is the same as Hieracium ×floribunda.
I am uncertain what the name of the hybrid should be in Pilosella.
This will have to remain an issue until I find the proper name.

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-02-14 13:31Z]

Found it: Pilosella floribunda (Wimm. & Grab.) Fr. in Fries & Lagger, Hierac. Eur. Exs.: [in schedis] Pilosella No. 17. 1862-18

Source: Source: Cichorieae portal - http://wp6-cichorieae.e-taxonomy.eu/portal/?q=cdm_dataportal/taxon/c05d834f-07cd-4876-adfe-dee79ec883bd/synonymy

This would have precedence over the Arvet-Touvet name (1880), if the publication is OK. I queried Gandhi and Zarucchi.
